### DISCUSSION TITLE
Recommendations for testing

### DISCUSSION THREAD

**Discussion by davidwinter** [AR10Dev/tauri-solid-ts-tailwind-vite#105] [Q&A] [answered]
I've used [AVA](https://github.com/avajs/ava) before, but I wouldn't be tied to it. But what is the recommendation for testing, and configuring Typescript to be able to write tests with it, and to run the tests? Would appreciate any suggestions.

**Comment by AR10Dev** [comment, marked as answer]
Hi, I usually follow these two guides when writing tests for a Tauri app: [this one](https://yonatankra.com/how-to-setup-vitest-in-a-tauri-project/) and [this one](https://yonatankra.com/how-to-write-unit-tests-in-rust-for-tauri/). I like using [Vitetest](https://vitest.dev/), so that's my go-to.
